requirements-dev.txt

```diff
@@ -1,17 +1,19 @@
-black==19.10b0
+--index-url https://pypi.devinfra.sentry.io/simple
```

Comment:
Let's leave a comment here referring to the README for how to add new dependencies.